src/async_wrap-inl.h

```diff
@@ -26,6 +26,7 @@
 
 #include "async_wrap.h"
 #include "base_object-inl.h"
+#include "node_errors.h"
```

Comment:
As I mentioned out earlier, this is unused:

```suggestion
```

Reply:
@addaleax, thanks. Missed it in previous commit.